src/pages/weatherstate/weatherstates.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import State from './weatherstate';
+import Weather from '../weatherpage/weather';
+import './weatherstates.css';
+
+const States = () => {
+  const location = useLocation();
+  const countryName = location.state.name;
+  const countryState = useSelector((state) => (state.countries[countryName]));
+  return (
+    <div className="statesPage" data-testid="statesContent">
+      <div className="weatherInfo">
+        <Weather
+          name={countryName}
+        />
+      </div>
+      <div className="padding chooseState">
+        <h2 className="states-text">
+          States
+        </h2>
+        <p className="states-prag">
+          Choose a state:
+          {' '}
+          {countryState.states.length}
+          {' '}
+          States
+        </p>
+      </div>
+      <div className="states padding">
+        {countryState.states.map((state) => (
+          <State
+            name={state.name}
+            key={state.name}
+            code={state.state_code}
+          />
+        ))}
+      </div>
```

Comment:
[REQUIRED] One of the aims of this project is to teach you to follow client requirements. Kindly ensure your app design is close to Nelson's template in terms. of typography and layout

![image](https://user-images.githubusercontent.com/62938735/213435276-59d08227-4ffb-4ac0-b67f-57f1f756f539.png)